checkers/Board.py

```diff
@@ -90,10 +90,20 @@ def find_valid_moves(self, player_turn):
                         self.grid[curr_row][curr_col].valid_grids.append((right_grid_row_, right_grid_col_))
                         self.opponent_piece_eaten['right'] = [right_grid_row, right_grid_col, right_grid_row_, right_grid_col_]
 
-            if self.grid[right_grid_row][right_grid_col] == 0:
+            else:
                 self.grid[curr_row][curr_col].valid_grids.append((right_grid_row, right_grid_col))
 
 
+        if 'left' in self.opponent_piece_eaten and 'right' not in self.opponent_piece_eaten:
+            print("Left side exist and right not")
+            self.grid[curr_row][curr_col].valid_grids = self.grid[curr_row][curr_col].valid_grids[:1]
+
+        if 'right' in self.opponent_piece_eaten and 'left' not in self.opponent_piece_eaten:
+            print("Right side exist and left not")
+            self.grid[curr_row][curr_col].valid_grids = [self.grid[curr_row][curr_col].valid_grids[-1]]
+            print(self.grid[curr_row][curr_col].valid_grids)
+
+
```

Comment:
added feature of only showing eating move. @aungmcs

Reply:
The bug still there when there are more than one possible capture moves
![Screen Shot 2021-01-08 at 11 44 57 AM](https://user-images.githubusercontent.com/24309297/103977322-6cb32480-51a7-11eb-8a7e-1515460ca150.png)
.